lib/AST/ASTContext.cpp

```diff
@@ -215,7 +215,10 @@ FOR_KNOWN_FOUNDATION_TYPES(CACHE_FOUNDATION_DECL)
   /// func _stdlib_isOSVersionAtLeast(Builtin.Word,Builtin.Word, Builtin.word)
   //    -> Builtin.Int1
   FuncDecl *IsOSVersionAtLeastDecl = nullptr;
-  
+
+  /// func _stdlib_enumerateCaseValues<Enum: ValueEnumerable>(Enum.Type) -> [T]
```

Comment:
nit: should be `-> [Enum]`. How did you choose this name? To me `enumerate` sounds like a verb here, but I would expect a noun. Perhaps `enumeration` or `enum`? or even a `get` prefix?

Reply:
`enumerate` is the operative verb, as in “to determine the number of”.  Good catch.

Reply:
You can decide whether the [API Design Guidelines](https://swift.org/documentation/api-design-guidelines/) apply here, but:

> **Name functions and methods according to their side-effects**
> Those without side-effects should read as noun phrases, e.g. `x.distance(to: y)`, `i.successor()`.